docs/docs/configuration/schema-repository.md

```diff
@@ -27,24 +29,29 @@ Set schema repository type to `schema_repo` when using this schema repository.
 
 ## Configuration
 
-### Frontend and Consumers
-
-Frontend and Consumers module share the same configuration options. To enable schema-repo, set:
```

Comment:
"To enable schema-repo, set" could be confusing because we don't have any flag to enable or disable the schema repository.